sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/ColumnarTestUtils.scala

```diff
@@ -51,6 +51,8 @@ object ColumnarTestUtils {
       case DOUBLE => Random.nextDouble()
       case STRING => UTF8String.fromString(Random.nextString(Random.nextInt(32)))
       case BINARY => randomBytes(Random.nextInt(32))
+      case CALENDAR_INTERVAL =>
+        new CalendarInterval(Random.nextInt(), Random.nextInt(), 0)
```

Comment:
any reason to assign 0 here?

Reply:
nope, just for brevity, and the microseconds part is also covered by other tests.

Reply:
Shall we use `Random.nextLong()` for consistency?
Also, cc @gengliangwang since this is random data generation.